src/main/java/tasks/Tasks.java

```diff
@@ -0,0 +1,156 @@
+                System.out.println("Number does not exist in list, please try again.");
+            } catch(NumberFormatException e){
+                System.out.println("Please add 'todo', 'deadline' or 'event' in the front of your task!");
+            }
+
+        } else if(input.toLowerCase().startsWith("delete") && input.split(" ").length == 2) {
+            try {
+                int taskIndex = Integer.parseInt(input.split(" ")[1]) - 1;
+                printDeleted(tasksAL, taskIndex);
+                tasksAL.remove(taskIndex);
+            } catch(IndexOutOfBoundsException e){
+                System.out.println("Number does not exist in list, please try again.");
+            } catch(NumberFormatException e){
+                System.out.println("Please enter delete followed by the number of the task you would like deleted.");
+            }
+
+        } else if (input.toLowerCase().startsWith("todo")) {
+            if (input.split(" ").length == 1) {
+                System.out.println("Please add a task after 'todo'");
+            } else {
+                tasksAL.add(new TodoTasks(input.substring(5)));
+                printAdded(tasksAL);
+            }
+
+        } else if (input.toLowerCase().startsWith("deadline")) {
+            try {
+                tasksAL.add(new DeadlineTasks(extractTask(input, "deadline"), extractDateTime(input)));
+                printAdded(tasksAL);
+            } catch (StringIndexOutOfBoundsException e){
+                System.out.println("Please use / followed by a deadline");
+            }
+
+        } else if (input.toLowerCase().startsWith("event")) {
+            try {
+                tasksAL.add(new EventTasks(extractTask(input, "event"), extractDateTime(input)));
+                printAdded(tasksAL);
+            } catch (StringIndexOutOfBoundsException e){
+                System.out.println("Please use / followed by a date or time");
+            }
+
+        } else {
+            System.out.println("Please add 'todo', 'deadline' or 'event' in the front of your task!");
+        }
+        try {
+            FileWriter myWriter = new FileWriter(saveFilesDir); //directory
+            for (Tasks task : tasksAL) {
+                myWriter.write(task.getTaskData());
+                myWriter.write(System.lineSeparator());
+            }
+            myWriter.close();
+        } catch (IOException e) {
+            System.out.println("An error occurred.");
+        }
+    }
+
+    public static void loadTasks(String input, ArrayList<Tasks> tasksAL) {
+        String[] data = input.split(",");
+        switch (data[0]){
+        case "T":
+            tasksAL.add(new TodoTasks(data[2]));
+            break;
+        case "D":
+            tasksAL.add(new DeadlineTasks(data[2], data[3]));
+            break;
+        case "E":
+            tasksAL.add(new EventTasks(data[2], data[3]));
+            break;
+        default:
+            checkComplete(data[1], tasksAL);
```

Comment:
what does data[0],data[2] andddata[3] stands for? perhaps change data[0] -> taskType and other to improve readability?